### ISSUE TITLE
Modify: HP:0011170(Generalized myoclonic-atonic seizure)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5672]
HP:0011170(Generalized myoclonic-atonic seizure)\
Parent: HP:0032677(Generalized-onset motor seizure)\
Add missing relation: Included in HPO.terms.to.correct_2020-04-30 as a suggested correction

**Comment by pnrobinson** [comment]
This is a grand child of Generalized-onset motor seizure and so we cannot add this link.
https://hpo.jax.org/app/browse/term/HP:0002123

**Comment by DLewisSmith** [comment]
Hi Peter, in the file HPO.terms.to.correct_2020-04-30 I have tried to indicate that the current relationship between HP:0011170 (Generalized myoclonic-atonic seizure) and HP:0002123 (Generalized myoclonic seizure) should be deleted (in the 'advice' column), and replaced by a relationship in which HP:0011170 (Generalized myoclonic-atonic seizure) becomes a child of HP:0032677 (Generalized-onset motor seizure). So as you have observed, it ceases to be a grandchild and becomes a child. This is because the ILAE does **not** currently consider Myoclonic-atonic seizures to be a form of Myoclonic seizure but both are children of Motor seizure. I have tried to use this structure consistently in corrections file to indicate the changes needed. The other file (Seizure.relationships.to.correct_2020-04-30) includes these changes and some additional parent-child relationships that appear to be missing. Hope this is clear, happy to explain further by call or message.
